src/components/hud-controller.js

```diff
@@ -34,11 +37,14 @@ AFRAME.registerComponent("hud-controller", {
     const pitch = head.rotation.x * THREE.Math.RAD2DEG;
     const yawDif = deltaAngle(head.rotation.y, hud.rotation.y) * THREE.Math.RAD2DEG;
 
+    // HUD is always visible until first hover, to increase discoverability.
+    const forceHudVisible = !this.store.state.activity.hasHoveredInWorldHud;
```

Comment:
Don't need to make a name for this value -- just use in the `if`'s predicate directly

Reply:
this is "variable name as documentation" :)